test/scalability/mzcompose.py

```diff
@@ -49,7 +49,7 @@
     Postgres(),
 ]
 
-REGRESSION_THRESHOLD_AS_PERCENT_DECIMAL = 0.2
+DEFAULT_REGRESSION_THRESHOLD_AS_PERCENT_DECIMAL = 0.2
```

Comment:
Giving you also the chance to challenge the name again 😀

Reply:
The name still makes no sense to me since nothing is seen as a "part of every hundred"

Reply:
Any alternative suggestions?

Reply:
I don't think it is clearer but I will adjust it.